processor/processor.go

```diff
@@ -179,19 +194,143 @@ func (proc *HandleT) Setup(backendConfig backendconfig.BackendConfig, gatewayDB
 	}
 }
 
+type StoreErrorOutputT struct {
```

Comment:
Let's move this backing up logic into a separate file in the same package.

Reply:
done